vtex-apps/login
# Feature/disabled email inputs [VID-117]

#### What problem is this solving?
This adds `disabledEmailInputs` prop to disable editing in email fields. This will be used by checkout to lock a known user

#### How to test it?

Go into
https://rafaprtest5--storecomponents.myvtex.com/
and click `Sign in`, then `Receive access code by email` or `Sign in with email and password`. You'll see the email input disabled.
If you add the query string `?userEmail=example@email.com`, you can lock a specific email.

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/22064061/90700883-51c62f00-e25d-11ea-9160-75e5ac003480.png)
